src/models/deleteParkingLot.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+export async function deleteParkingLot(chosenLot) {
+  try {
+      if (chosenLot.chosenLotID === '' || chosenLot.chosenLotID === undefined) {
+        console.warn("got empty ID or name for destination parking lot")
+        throw new Error('Please choose destination')
+      }
+      const response = await axios.delete(process.env.BASE_API_URL+`/parking-lots/${chosenLot.chosenLotID}`,{
+        headers: {
+          'Content-Type': 'application/json',
+          'Authorization': `Bearer ${sessionStorage.getItem('token')}`
+        }
+      });
+      console.debug("Got response data for deleting a parking lot", response.data)
+      return response.data;
+    } catch (error) {
+      if (error.response) {
```

Comment:
Let's improve error handling like you did in getAllAdminParkLots:
```
catch (error) {
        if (error.response) {
            console.error('Failed to ...', error.response);
            throw new Error(error.response.data || 'an error occurred')
        } else if (error.request) {
            console.error('Didnt receive response after making request ...', error.request);
            throw new Error('no healthy upstream')
        } else {
            console.error('Failed to setup request to ...', error.message);
            throw new Error('an error occurred')
        }
    }
```

I don't remember if you had that at the very start and I told you to not do it like so - if so, I was wrong sorry,[ axios docs](https://axios-http.com/docs/handling_errors) show how to handle errors